_episodes/02-make-it-public.md

```diff
@@ -58,21 +58,46 @@ on this topic, and other advice on starting an open source project, check out th
+ 
 > ## Challenge: Improve the README
 >
 > The README.md file that GitHub generated when we created the project is quite
 > boring. Add some more content using some of the Markdown syntax introduced
-> above. You could add a few sentences about the purpose of your software, some
+> above (if you don't remember a Markdown command you can refer to the 
+> [cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) mentioned above). 
+> You could add a few sentences about the purpose of your software, some
 > details of the author(s) and their affiliation(s), or anything else that you
 > think is important.
+
+{: .challenge}
+
+In addition to the README.md file, a good software project should include other "recommended community health files". What other files should you include in your project? You may find insights on health files to include in your project (to ensure that it meets the recommended community standards) in the Community Profile of the public repository hosting your project. Indeed, a public repository Community Profile provides rules and best practices on how to help grow a community and support contributors in using and contributing to your project. If your project is hosted on GitHub, you may refer to the [GitHub Community Profile](https://github.com/SoftDev4Research/4OSS-lesson/community), which can be accessed following [these instructions](https://help.github.com/en/articles/accessing-a-projects-community-profile) and where you can find a checklist that can be used to see if your project includes community health files recommended by GitHub. 
+These include a project DESCRIPTION file, a README file and a CODE_OF_CONDUCT, which contains standards on how to engage in the community. Furthermore, a project should have a CONTRIBUTING file (for details see episode on "Define clear and transparent contribution, governance and communication processes") outlining guidance for contributing to the project, and a LICENCE file that states how the code can be used. You should have a good idea of what is the appropriate license for your project after the [next episode](https://softdev4research.github.io/4OSS-lesson/03-use-license/index.html). Templates for GitHub issues and pull requests are also required for a complete community profile (see the [Software Carpentry lesson on Version Control with Git](http://swcarpentry.github.io/git-novice/) to learn about GitHub issues and pull requests).
+
+You might also want to add details of how to correctly cite the software after publication. This can be done by adding a file called CITATION.cff in the root of your repository (where other health files are stored). The .cff (Citation File Format) is a machine-readble and human-readble and -writable format providing citation information for research software.
```

Comment:
```suggestion
You should also add description on how others can correctly cite the software. This can be done by adding a file called CITATION.cff in the root of your repository (where other health files are stored). The .cff (Citation File Format) is a machine-readble and human-readble and -writable format providing citation information for research software.
```